pom.xml

```
@@ -62,8 +62,7 @@
   </scm>
 
   <properties>
-    <revision>4.3.17</revision>
-    <changelist>-SNAPSHOT</changelist>
+    <changelist>999999-SNAPSHOT</changelist>
```

if we want to keep the leading 4 as per the previous suggestion

```suggestion
    <revision>4</revision>
    <changelist>999999-SNAPSHOT</changelist>
```